src/gkApi.ts

```diff
@@ -41,6 +41,30 @@ const getAccessToken = async () => {
 	return cookie?.value;
 };
 
+export const checkin = async () => {
+	const token = await getAccessToken();
+	if (!token) {
+		return;
+	}
+
+	// Don't check in more than once every 12 hours to reduce amount of requests
+	const { lastCheckin } = await storage.local.get('lastCheckin');
+	if (lastCheckin && Date.now() - lastCheckin < 1000 * 60 * 60) {
+		return;
```

Comment:
1: I think this currently checks every hour
2: is there a reason this is different than the gkdev checkin that's once every 24 hours?

Reply:
Good catch.
And I went with 12 instead of 24 for cases such as using it at 10am the first day and 9am the next.